grails-app/controllers/org/pih/warehouse/api/ApiController.groovy

```diff
@@ -209,6 +209,7 @@ class ApiController {
                 timezone             : timezone,
                 minimumExpirationDate: minimumExpirationDate,
                 activeLanguage       : currentLocale.toString(),
+                activeLanguageTag    : currentLocale.toLanguageTag(),
```

Comment:
I assume this is done, so that in case of `es-MX`, the `toLanguageTag()` would return `es`, whereas `toString()` was returning `es-MX`?

Reply:
No. The language tag will be `es-MX` for Mexican Spanish, and this is required to be able to use it in the js ```Number.toLocaleString()``` or in the `moment.js` in that format.
toString returns `es_MX` which cannot be used in `toLocaleString`